### PR TITLE
Actualized todos and fixmes [ECR-1554]

### PR DESCRIPTION
**Pull request by popzxc**
What have been done:

1. All the TODOs and FIXMEs without ticket number were either marked with created issue or removed.
2. All existing issues for FIXMEs and TODOs were checked:
- If issue was marked as `Done`, TODO was removed.
- If issue was marked as `To do`, it was set as related to `ECR-1554`.
3. Style of TODOs and FIXMEs was unified, now it looks like: `TODO: Issue description. (ECR-xxxx)`

What have NOT been done:
1. Actualization of existing tasks. I suppose that whoever will be assigned to activity will check things himself.
2. I did not correct any grammar in existing TODOs (it've already taken too much time, grammar checks would have make this activity too big).

### CODE REVIEW

File: exonum/src/blockchain/mod.rs [hunk 1]
```diff
@@ -221,7 +221,7 @@ impl Blockchain {
             {
                 let mut schema = Schema::new(&mut fork);
                 if schema.block_hash_by_height(Height::zero()).is_some() {
-                    // TODO create genesis block for MemoryDB and compare in hash with zero block
+                    // TODO create genesis block for MemoryDB and compare in hash with zero block. (ECR-1630)
```
Comment (aleksuss) [inline exonum/src/blockchain/mod.rs]: `...compare it hash...`

File: exonum/src/events/network.rs [hunk 2]
```diff
@@ -46,7 +46,7 @@ pub enum NetworkRequest {
 
 #[derive(Serialize, Deserialize, Debug, Clone, Copy)]
 pub struct NetworkConfiguration {
-    // TODO: think more about config parameters (ECR-162)
+    // TODO: Rhink more about config parameters. (ECR-162)
```
Comment (aleksuss) [inline exonum/src/events/network.rs]: Rhink -> Think ?

File: exonum/src/blockchain/mod.rs (hunk 1 above)
Comment (popzxc) [reply exonum/src/blockchain/mod.rs]: Another remark: I did not correct any grammar in existing TODOs (it've already taken too much time, grammar checks would have make this activity too big).
But if you asking me to (:

File: .travis.yml [hunk 3]
```diff
@@ -90,7 +90,7 @@ jobs:
     - find . -not -path "./3rdparty/*" -and -not -path "./node_modules/*" -name "*.md" | xargs ./node_modules/.bin/cspell
     - find . -not -path "./3rdparty/*" -and -not -path "./node_modules/*" -name "*.md" | xargs ./node_modules/.bin/markdownlint --config .markdownlintrc
     - cargo doc --no-deps
-    # TODO: a tmp hack to ignore warnings about missing pages [ECR-703]
+    # TODO: Temporary hack to ignore warnings about missing pages. (ECR-703)
```
Comment (DarkEld3r) [inline .travis.yml]: :+1:

**Pull request merged**